src/Action.hs

```diff
@@ -278,11 +278,9 @@ getSpacesForCurrentUser = do
     else
         pure []
 
--- TOOD: Implement it
+-- FIXME: Authorization
```

Comment:
I would leave that FIXME.  We could start adding that all over the place and still miss places, so IMO it is more distracting than helpful.